include/SDL3/SDL_gpu.h

```diff
@@ -2257,6 +2260,123 @@ extern SDL_DECLSPEC const char * SDLCALL SDL_GetGPUDeviceDriver(SDL_GPUDevice *d
  */
 extern SDL_DECLSPEC SDL_GPUShaderFormat SDLCALL SDL_GetGPUShaderFormats(SDL_GPUDevice *device);
 
+/**
+ * Returns a property group containing read-only debug information associated
+ * with this GPU context. All properties are optionally provided and may differ
+ * between GPU backends and SDL versions.
+ *
+ * The following properties are provided by SDL:
+ *
+ * ```
+ * SDL_PROP_GPU_DEVICE_DEBUG_NAME_STRING:
+ *   Contains the name of the underlying device as reported by the system
+ *   driver. This string has no standardized format, is highly inconsistent
+ *   between hardware devices and drivers, and is able to change at any time. Do
+ *   not attempt to parse this string as it is bound to fail at some point in
+ *   the future when system drivers are updated, new hardware devices are
+ *   introduced, or when SDL adds new GPU backends or modifies existing ones.
+ *
+ *   Strings that have been found in the wild include:
```

Comment:
This doesn't need to be this large of a list, it can be shorter with some different representative examples.

Also, for wiki formatting it should be something like this:
```
 *   Strings that have been found in the wild include:
 * 
 * - GTX 970
 * - GeForce GTX 970
etc...
```

Reply:
I tried to keep the lists as short as possible but I wanted to show how wide a variety the strings can be in the wild to help convince app developers to not parse the strings. The list shows the same device multiple times in different formats, it shows devices from multiple manufacturers, it shows oddball devices that developers may not be familiar with, it shows software implementations of the system APIs, and it shows "wrapped" devices going through some system emulation API.

Reply:
I've reformatted this documentation, pruned the lists slightly, and added a paragraph explaining the patterns between the device strings.